### DISCUSSION TITLE
Fragment matcher json without possible types

### DISCUSSION THREAD

**Discussion by electerious** [dotansimha/graphql-code-generator#5835] [Q&A]
Hi, I'm generating a `possibleTypes.json` with the `fragment-matcher` plugin. The JSON however doesn't contain the possible types directly and instead has another key called `possibleTypes`.

What I currently get:

```json
{
  "possibleTypes": {
    "BooleanTreeNode": [
      "OperatorNode",
      "StatementNode"
    ]
  }
}
```

What I want:

```json
{
  "BooleanTreeNode": [
    "OperatorNode",
    "StatementNode"
  ]
}
```

Is is possible to get a JSON that contains the possible types directly? This would make the import a lot nicer (it's currently `import possibleTypes from '@repo/possibleTypes'` and I get the types via `possibleTypes.possibleTypes`.

**Comment by ardatan** [comment]
It's not possible currently. Would you create a PR for that feature?
